packages/manager/modules/manager-pci-common/src/api/data/feature-availability.ts

```diff
@@ -0,0 +1,25 @@
+import apiClient from '@ovh-ux/manager-core-api';
+
+export type FeatureAvailabilityQueryParams = Partial<{
+  app: string;
+}>;
+
+export type FeatureAvailabilityResponse<T extends string = string> = {
+  [Key in T]?: boolean;
+} & {
+  /* @deprecated */
+  feature: string;
+  /* @deprecated */
+  result: boolean;
+};
+
+export const getFeatureAvailability = <T extends string = string>(
```

Comment:
i don't think this is specific to pci common this should belong in MRC

Reply:
We can't add more features to MRC in ODS17